srcpkgs/etesync-dav/template

```diff
@@ -1,22 +1,16 @@
 # Template file for 'etesync-dav'
 pkgname=etesync-dav
-version=0.20.4
-revision=3
+version=0.32.1
+revision=1
+archs="x86_64"
```

Comment:
Why was this marked x86_64 only?

Reply:
not sure how I missed this. (same question for etesync-base)